exchanges/binance/binance.go

```diff
@@ -269,6 +269,42 @@ func (b *Binance) GetHistoricalTrades(symbol string, limit int, fromID int64) ([
 	return resp, b.SendHTTPRequest(path, &resp)
 }
 
+// GetTradesBetween returns aggregated trade activity between two timestamps
+//
+// symbol: string of currency pair
+// start: epoch timestamp of beginning of query
+// end: epoch timestamp of end of query
+func (b *Binance) GetTradesBetween(symbol string, start, end time.Time) ([]AggregatedTrade, error) {
+	collectedTrades := []AggregatedTrade{}
+	for {
+		queryEnd := start.Add(1 * time.Hour)
+		if start.Equal(end) {
```

Comment:
If the start time equals the end time, could throw an error stating this to the user.